.circleci/config.yml

```diff
@@ -0,0 +1,144 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: docker.io/openwrtorg/packages-cci:v1.0.2
+    environment:
+      - SDK_HOST: "downloads.openwrt.org"
+      - SDK_PATH: "snapshots/targets/arx71xx/generic"
+      - SDK_FILE: "openwrt-sdk-18.06.1-ar71xx-generic_*.Linux-x86_64.tar.xz"
```

Comment:
Is there a reason to specify the ancient 18.06.1 as the SDK version? Why not the recent 18.06.4 or the snapshot-based ongoing SDK from buildbot: 
https://downloads.openwrt.org/releases/18.06-SNAPSHOT/targets/ar71xx/generic/openwrt-sdk-18.06-SNAPSHOT-ar71xx-generic_gcc-7.3.0_musl.Linux-x86_64.tar.xz
as generically
openwrt-sdk-18.06*-ar71xx-generic_*.Linux-x86_64.tar.xz

I noticed that the 8 month old 18.06 packages repo circleci definition seems to contain 18.06.1, but that is likely just an oversight, or how? @thess  @champtar